main/main.go

```diff
@@ -162,11 +164,23 @@ func reportExecutionStatus() {
 
 func install() {
 	operation := "install"
+	path, _ := os.Getwd()
+	_, err := exec.Command("/bin/sh", filepath.Join(path, "scripts/service_install.sh")).Output()
```

Comment:
With these changes, the new service will be installed every time the extension is called. Wouldn't it be a better design to kind of parameterize it? Like basically read some properties from the settings file and based on that install/enable/delete the service?

That would preserve the original functioning of the extension too and no existing scenarios would have to be modified for it.